### PR TITLE
Add RSS XML

### PR DESCRIPTION
**Pull request by dNitza**
This PR adds an RSS feed at `/feed.xml`.

### CODE REVIEW

File: apps/main/lib/main/views/pages/feed.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        include Main::Import("main.persistence.repositories.posts")
+
+        def call(options = {})
+          template_path = "#{config.root}/pages/feed.#{options[:format]}.#{options[:engine]}"
+          Tilt::BuilderTemplate.new(template_path).render( posts: Decorators::PublicPost.decorate(posts.all) )
+        end
+      end
+    end
+  end
+end
```
Comment (dNitza) [inline apps/main/lib/main/views/pages/feed.rb]: I'm not certain that my approach to creating a new view to render XML is correct.

File: apps/main/lib/main/persistence/repositories/posts.rb [hunk 2]
```diff
@@ -25,6 +25,14 @@ def listing(page: 1, per_page: 20)
             .as(Entities::Post::WithAuthor)
         end
 
+        def all
```
Comment (dNitza) [inline apps/main/lib/main/persistence/repositories/posts.rb]: I'm also not sure if we want `all` posts for the RSS feed or the last `50`, say. If we only want a limited set, we could probably rename this to `for_rss_feed`.

File: apps/main/web/routes/pages.rb [hunk 3]
```diff
@@ -25,5 +25,10 @@ class Application < Dry::Web::Application
       end
       r.view "pages.work"
     end
+
+    route "feed.xml" do |r|
```
Comment (dNitza) [inline apps/main/web/routes/pages.rb]: Not sure if this is the correct way to specify a format.

File: apps/main/lib/main/views/pages/feed.rb (hunk 1 above)
Comment (timriley) [inline apps/main/lib/main/views/pages/feed.rb]: If it works, I think this is good enough for now. Later on we can update dry-view to support layout-less views.

File: apps/main/lib/main/persistence/repositories/posts.rb (hunk 2 above)
Comment (timriley) [inline apps/main/lib/main/persistence/repositories/posts.rb]: I think the practice is just to return the latest `n` posts. 20 would be fair.

File: apps/main/web/routes/pages.rb (hunk 3 above)
Comment (timriley) [inline apps/main/web/routes/pages.rb]: It's really just a URL. Seems fine to me :)

The alternative would be just to offer `/feed` as the URL, which is pretty common too? @makenosound, do you have opinions?

Comment (makenosound) [inline apps/main/web/routes/pages.rb]: I have no strong feelings one way or the other. Keeping it formatless like `/feed` seems fine to me.

Comment (dNitza) [inline apps/main/web/routes/pages.rb]: Cool, I shall remove the `.xml` and merge

**Pull request merged**